tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -20,6 +20,14 @@ function HeaderComp() {
   const pathname = useSelector((state) => state.router.location.pathname)
   const user = useSelector((state) => state.auth.user)
 
+  const isSystemAdmin = () => {
+    if (user && user.roles) {
+      return user.roles.some((role) => role.name === 'System Admin')
```

Comment:
Do we need to manually add this Group? By default I have `OFA Admin`

cc @carltonsmith

Reply:
I had updated this PR to use the `OFA Admin` role but it seemed those requirements had changed?

Reply:
@jtwillis92 @jorgegonzalez Yes, locally you can add the group via Django Admin. That's what I did on the dev site.